core/domain/stats_jobs_continuous_test.py

```diff
@@ -452,7 +451,7 @@ def test_one_answer(self):
             # get job output of first state and check it
             calc_output_domain_object = (
                 stats_jobs_continuous.InteractionAnswerSummariesAggregator.get_calc_output(
-                    exp_id, exp_version, FIRST_STATE_NAME, calc_id))
+                    exp_id, FIRST_STATE_NAME, calc_id, exp_version))
```

Comment:
Do you have a strong preference for scrapping the default? I updated the tests; I generally think the default case is 'all', so I think keeping it is reasonable.